code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -33,23 +33,91 @@
 	var/next_dest
 	var/next_dest_loc
 
+	var/obj/item/weapon
+	var/weapon_orig_force = 0
+
+	var/list/stolen_valor
+
+
+	var/orig_name
+
+	var/list/officers = list("Admiral", "Spec Ops Officer", "Captain", "Head of Personnel", "Death Commando", "Head of Security")
```

Comment:
You get the title to use from the `assigned_role` var but `Admiral` and `Death Commando` are never used as `assigned_role`, they are only used on ID cards. `Spec Ops Officer` just doesn't appear anywhere in the codebase. So they're pointless to include.

Reply:
I realized that, but admins can still manually set them for gimmicks, and I might make setting those outfits give those roles later on. Thank you for pointing out that it's Special Ops Officer instead of Spec Ops Officer though!

Reply:
If it's not going to be used then add these titles in another PR.

Reply:
Done